apps/condo/domains/notification/tasks/deliverMessage.js

```diff
@@ -171,18 +171,20 @@ async function deliverMessage (messageId) {
         processingMeta.transport = transport
 
         try {
-            const isAllowed = get(userTransportSettings, transport)
-            if (isAllowed === false) {
-                transportMeta.status = MESSAGE_DISABLED_BY_USER_STATUS
-            } else {
-                const adapter = TRANSPORT_ADAPTERS[transport]
-                // NOTE: Renderer will throw here, if it doesn't have template/support for required transport type.
-                const messageContext = await adapter.prepareMessageToSend(message)
+            const adapter = TRANSPORT_ADAPTERS[transport]
+            // NOTE: Renderer will throw here, if it doesn't have template/support for required transport type.
+            const messageContext = await adapter.prepareMessageToSend(message)
 
-                processingMeta.messageContext = messageContext
-                transportMeta.messageContext = messageContext
-                processingMeta.transports.push(transport)
+            processingMeta.messageContext = messageContext
+            transportMeta.messageContext = messageContext
+            processingMeta.transports.push(transport)
 
+            const isAllowedByUser = get(userTransportSettings, transport)
+            if (isAllowedByUser === false) {
+                transportMeta.status = MESSAGE_DISABLED_BY_USER_STATUS
+                processingMeta.step = MESSAGE_SENT_STATUS
+                successCnt++
```

Comment:
Why is there an abbreviation?
```suggestion
                successCount++
```

Reply:
This is not a variable of mine :)
I decided to keep the name as is.